massa-node/base_config/config.toml

```diff
@@ -1,96 +1,175 @@
     wakeup_interval = 5000
+    # path to the cold stored peers
     peers_file = "storage/peers.json"
+    # path to the initial peer file
     initial_peers_file = "base_config/initial_peers.json"
+    # target number of connection to a bootstrap node, once the bootstrap succeeded
     target_bootstrap_connections = 1
+    # max number of connection attempts to bootstrap nodes, once the bootstrap succeeded
     max_out_bootstrap_connection_attempts = 1
+    # target number of non bootstrap out connections
     target_out_nonbootstrap_connections = 10
+    # max number of in non bootstrap connections
     max_in_nonbootstrap_connections = 15
+    # max number of in connections per ip
     max_in_connections_per_ip = 5
+    # max number of out non bootstrap connection attempts
     max_out_nonbootstrap_connection_attempts = 10
+    # max number of stored idle peers
     max_idle_peers = 10000
+    # max number of stored banned peers
     max_banned_peers = 100
+    # max number of advertized peers
     max_advertise_length = 10000
+    # peers are cold stored every peers_file_dump_interval millis
     peers_file_dump_interval = 10000
+    # max size of sent messages
     max_message_size = 1048576000
+    # after message_timeout millis we are no longer waiting on handshake message
     message_timeout = 5000
+    # Every ask_peer_list_interval in millis we ask every one for its advertisable peers list.
     ask_peer_list_interval = 30000
+    # path to the node private key (not the staking key)
     private_key_file = "config/node_privkey.key"
+    # max number of asked blocks per message
     max_ask_blocks_per_message = 128
+    # max number of operation per message
     max_operations_per_message = 1024
+    # max number of endorsements per message
     max_endorsements_per_message = 1024
+    # max millis to wait while sending an event before dropping it
     max_send_wait = 500
+    # we forget we banned a node after ban_timeout
     ban_timeout = 3600000
 
 [bootstrap]
+    # list of bootstrap (ip, public key)
     bootstrap_list = [
         ["149.202.86.103:31245", "5GcSNukkKePWpNSjx9STyoEZniJAN4U4EUzdsQyqhuP3WYf6nj"],
         ["149.202.89.125:31245", "5wDwi2GYPniGLzpDfKjXJrmHV3p1rLRmm4bQ9TUWNVkpYmd4Zm"],
         ["158.69.120.215:31245", "5QbsTjSoKzYc8uBbwPCap392CoMQfZ2jviyq492LZPpijctb9c"],
         ["158.69.23.120:31245", "8139kbee951YJdwK99odM7e6V3eW7XShCfX5E2ovG3b9qxqqrq"]
     ]
+    # [optionnal] for bootstrap servers to port people will bootstrap from
     bind = "[::]:31245"
+    # timeout to establish a bootstrap connection
     connect_timeout = 15000
+    # delay before an other bootsrap attempt
     retry_delay = 5000
+    # if ping is too high bootstrap will be inturrupted
     max_ping = 10000
+    # after read timeout millis, we stop reading
     read_timeout = 10000
+    # after write_timeout millis, we stop writing
     write_timeout = 10000
+    # if enabled, time will be synchronized on the server time
```

Comment:
when enabled, apply a correction to the local computer clock to match bootstrap server time